torch/csrc/jit/frontend/environment.h

```diff
@@ -0,0 +1,384 @@
+#include <torch/csrc/jit/frontend/ir_emitter_utils.cpp>
```

Comment:
The fact that we have to include an implementation file usually indicates bigger problems. What we're trying to accomplish by including a cpp file instead of an h file?

Reply:
Good point, `ir_emitter_utils` should be a header file since it can compile independently